.circleci/config.yml

```diff
@@ -18,6 +21,20 @@ references:
   study_guids: &study_guids
     "basil NA CMI-OSTEO cmi-brain ANGIO cmi-mbc testboston cmi-mpc PRION atcp rarex RGP cmi-esc cgc circadia cmi-pancan singular brugada cmi-lms fon"
 
+  # Filter for PR branch only
+  filter-pr-branch: &filter-pr-branch
+    filters:
+      tags:
+        ignore: /.*/
+      branches:
+        ignore: develop
+
+  # Filter for develop branch only
+  filter-develop-branch: &filter-develop-branch
+    filters:
+      branches:
+        only: develop
```

Comment:
What is that (`filter-pr-branch`, `filter-develop-branch`) for?
I was not able to find its using.

Reply:
Yes, earlier changes used them but have been removed. I'll remove two filters.